src/appleseed/renderer/modeling/camera/fisheyelenscamera.h

```diff
@@ -0,0 +1,79 @@
+
+//
+// This source file is part of appleseed.
+// Visit https://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2014-2018 Jino Park, The appleseedhq Organization
```

Comment:
That line should be:

    // Copyright (c) 2018 Jino Park, The appleseedhq Organization